pkgs/development/libraries/libvirt/default.nix

```
@@ -152,6 +152,8 @@ stdenv.mkDerivation rec {
   ] ++ optionals stdenv.isDarwin [
     libiconv
     gmp
+    darwin.apple_sdk.frameworks.Carbon
+    darwin.apple_sdk.frameworks.AppKit
```

generally we would say 
```suggestion
    Carbon
    AppKit
```

and do a `inherit (darwin.apple_sdk.frameworks) Carbon AppKit;` in `all-packages.nix`

---

Done, thanks. I have seen it done both ways, so I went with this one because the base darwin set was already being brought into scope, but I will prefer this method from now on.

---

I generally prefer the "individual packages approach", as it makes overriding dependencies much easier.